.claude/skills/pr-address/SKILL.md

```
@@ -0,0 +1,91 @@
+kill $REST_PID
+pnpm types && pnpm lint && pnpm format
+```
+Never manually edit files in `src/app/api/__generated__/`.
+
+Then commit and **push immediately** — never batch commits without pushing.
+
+For backend commits in worktrees: `poetry run git commit` (pre-commit hooks).
+
+## The loop
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)